tests/cnt-i18n_xhtml-dir/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,22 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en" dir="rtl">
+  <head>
+    <title>Processing the 'dir' attribute in XHTML content</title>
+  </head>
+  <body>
+    <div dir="ltr">
+      <p>Test passes if:</p>
+      <ul>
+        <li>the numbered list below is adjusted to the right edge of the screen</li>
```

Comment:
"adjusted" is a weird verb here. Do you mean "the numbered list is right-aligned" (or "aligned right")?